temporal-sdk/src/test/java/io/temporal/workflow/versionTests/GetVersionWorkflowReplaceCompletelyTest.java

```diff
@@ -47,19 +49,21 @@
 
   @Test
   public void testGetVersionWorkflowReplaceCompletely() {
-    Assume.assumeFalse("skipping for docker tests", SDKTestWorkflowRule.useExternalService);
+    assumeFalse("skipping for docker tests", SDKTestWorkflowRule.useExternalService);
 
-    TestWorkflows.TestWorkflow1 workflowStub =
-        testWorkflowRule.newWorkflowStubTimeoutOptions(TestWorkflows.TestWorkflow1.class);
-    workflowStub.execute(testWorkflowRule.getTaskQueue());
+    TestWorkflow4 workflowStub =
+        testWorkflowRule.newWorkflowStubTimeoutOptions(TestWorkflow4.class);
+    assertTrue(workflowStub.execute());
   }
 
-  public static class TestGetVersionWorkflowReplaceCompletely
-      implements TestWorkflows.TestWorkflow1 {
+  public static class TestGetVersionWorkflowReplaceCompletely implements TestWorkflow4 {
+
+    private boolean hasReplayed;
 
     @Override
-    public String execute(String taskQueue) {
+    public boolean execute() {
       log.info("TestGetVersionWorkflow3Impl this=" + this.hashCode());
+      if (Workflow.isReplaying()) hasReplayed = true;
```

Comment:
`if (Workflow.isReplaying())` following by `if (!Workflow.isReplaying())` should be simplified into one `if` with `else`.